### ISSUE TITLE
Quicksoil should not form around clouds

### ISSUE THREAD

**Issue by MBatt1** [devs-immortal/The-Aether#39]
![image](https://user-images.githubusercontent.com/53229958/100687296-a3468580-3345-11eb-907b-b26359819411.png)

**Comment by Hephaestus-Dev** [comment]
I'll fix this when I add Gravitite ore (just waiting on the texture).

**Comment by CDAGaming** [comment]
Closed as Fixed from testing a bit tonight, feel free to make new issue if reoccurs.
